Formula/elasticsearch@2.4.rb

```diff
@@ -9,6 +9,8 @@ class ElasticsearchAT24 < Formula
 
   keg_only :versioned_formula
 
+  deprecate! date: "Februari 28, 2018"
```

Comment:
```suggestion
  deprecate! :date => "February 28, 2018"
```

(and same below). `brew style --fix homebrew/core` will fix these for you (but not the `February` typo)